dozer/cogs/starboard.py

```diff
@@ -0,0 +1,366 @@
+                return
+
+        # check if it's been cancelled in the starboard channel
+        if str(reaction) == config.cancel_emoji and msg.channel.id == config.channel_id:
+            db_msgs = await StarboardMessage.get_by(starboard_message_id=msg.id)
+            if len(db_msgs) and member.id == db_msgs[0].author_id:
+                DOZER_LOGGER.debug("Message cancelled in starboard channel, cancelling")
+                await self.remove_from_starboard(config, msg, True)
+                return
+
+        # check if it's been cancelled on the original message
+        if str(reaction) == config.cancel_emoji:
+            db_msgs = await StarboardMessage.get_by(message_id=msg.id)
+            if len(db_msgs) and member.id == db_msgs[0].author_id:
+                DOZER_LOGGER.debug("Message cancelled in original channel, cancelling")
+                try:
+                    starboard_msg = await self.bot.get_channel(config.channel_id). \
+                        fetch_message(db_msgs[0].starboard_message_id)
+                except discord.NotFound:
+                    DOZER_LOGGER.warning(f"Cannot find Starboard Message {db_msgs[0].starboard_message_id} to remove")
+                    starboard_msg = discord.Object(db_msgs[0].starboard_message_id)
+                await self.remove_from_starboard(config, starboard_msg, True)
+                return
+
+    @Cog.listener()
+    async def on_reaction_add(self, reaction, user):
+        """Pass the reaction add event onto our handler"""
+        await self.starboard_check(reaction, user)
+
+    @Cog.listener()
+    async def on_reaction_remove(self, reaction, user):
+        """Pass the reaction remove event onto our handler"""
+        await self.starboard_check(reaction, user)
+
+    @guild_only()
+    @group(invoke_without_command=True, aliases=['hof'])
+    async def starboard(self, ctx):
+        """Show the current server's starboard configuration.
+         A starboard (or a hall of fame) is a channel the bot will repost messages in if they receive a certain number\
+         of configured reactions.
+
+         To configure a starboard, use the `starboard config` subcommand.
+         """
+        config = await StarboardConfig.get_by(guild_id=ctx.guild.id)
+
+        if config:
+            await ctx.send(embed=self.make_config_embed(ctx, f"Starboard configuration for {ctx.guild}", config[0]))
+        else:
+            await ctx.send(f"This server does not have a starboard configured! See `{ctx.prefix}help starboard` for "
+                           f"more information.")
+
+    starboard.example_usage = """
+    `{prefix}starboard` - Get the current starboard settings for this server
+    """
+
+    @has_permissions(manage_guild=True, manage_channels=True)
+    @bot_has_permissions(add_reactions=True, embed_links=True)
+    @starboard.command()
+    async def config(self, ctx, channel: discord.TextChannel, star_emoji: typing.Union[discord.Emoji, str],
```

Comment:
This subcommand (and all the others) needs a `guild_only` check too; checks aren't applied recursively to child commands.